db/migrate/20241129052526_add_unique_index_to_collaborators.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniqueIndexToCollaborators < ActiveRecord::Migration[7.0]
```

Comment:
this we could have handled while CreateCollaborators migration only...

thats why I have mention multiple time always think of all the possible scenarios before adding a migration